pkg/security/ebpf/c/include/structs/events_context.h

```diff
@@ -65,7 +65,6 @@ struct cgroup_context_t {
 };
 
 struct container_context_t {
-    container_id_t container_id;
     struct cgroup_context_t cgroup_context;
 };
```

Comment:
Should we use directly `cgroup_context_t` instead of `container_context_t` now?

Reply:
I suggest to do this in a follow up PR as it will make the PR very noisy